FY2017/2017-01.beancount

```diff
@@ -4,204 +4,204 @@
 2017-01-01 open Income:Payout USD
 
```

Comment:
We could put these in FY2017.beancount since we are using this for the entire year and it is accounts that well will have used befor this fiscal year?